### PR TITLE
GC config knobs touch-ups

### LINKED ISSUE #16075: GC config knobs - two suggestions
Regarding https://docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector:

Feedback 1: I recommend adding an example of the config at the bottom, e.g. web.config, runtimeconfig.json (e.g. to answer "where in the JSON/.config does this go?")

Feedback 2: In the hex examples like "1D4C0", IMO "0x1D4C0" is a better/clearer recommendation

See https://twitter.com/Nick_Craver/status/1201686635043266561.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e544aba5-cfe6-1790-023e-bec738182ea8
* Version Independent ID: 8392a21a-8404-d352-ffb1-d6373bbbdd1b
* Content: [Garbage collector config settings](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector)
* Content Source: [docs/core/run-time-config/garbage-collector.md](https://github.com/dotnet/docs/blob/master/docs/core/run-time-config/garbage-collector.md)
* Product: **dotnet-core**
* GitHub Login: @gewarren
* Microsoft Alias: **gewarren**

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #16075 

@Maoni0 I didn't add the web.config examples that Nick Craver requested since this article is really about .NET Core. LMK if you think I should add them anyway.

[Internal preview link](https://review.docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector?branch=pr-en-us-16504)

### CODE REVIEW

File: docs/core/run-time-config/garbage-collector.md [hunk 1]
```diff
@@ -137,8 +209,20 @@ For more information about some of these settings, see the [Middle ground betwee
 | **runtimeconfig.json** | `System.GC.HeapHardLimit` | *decimal value* | .NET Core 3.0 |
 | **Environment variable** | `COMPlus_GCHeapHardLimit` | *hexadecimal value* | .NET Core 3.0 |
 
+Example:
+
+```json
+{
+   "runtimeOptions": {
+      "configProperties": {
+         "System.GC.HeapHardLimit": 80000
```
Comment (Maoni0) [inline docs/core/run-time-config/garbage-collector.md]: I know this is supposed to be an example only but it'd be good to have a more realistic value :) like 209715200 which is 200MiB.

File: docs/core/run-time-config/garbage-collector.md [hunk 2]
```diff
@@ -137,8 +209,20 @@ For more information about some of these settings, see the [Middle ground betwee
...
+      }
+   }
+}
+```
+
 > [!TIP]
-> If you're setting the option in *runtimeconfig.json*, specify a decimal value. If you're setting the option as an environment variable, specify a hexadecimal value. For example, to specify a heap hard limit of 80,000 bytes, the values would be 80000 for the JSON file and 13880 for the environment variable.
+> If you're setting the option in *runtimeconfig.json*, specify a decimal value. If you're setting the option as an environment variable, specify a hexadecimal value. For example, to specify a heap hard limit of 80,000 bytes, the values would be 80000 for the JSON file and 0x13880 or 13880 for the environment variable.
```
Comment (Maoni0) [inline docs/core/run-time-config/garbage-collector.md]: probably would the 80,000 bytes here too..sorry I should have pointed this out before

**Review by Maoni0 — CHANGES_REQUESTED** [review]
the rest looks great!

**Review by Maoni0 — APPROVED** [review]
LGTM!

**Review by Maoni0 — APPROVED** [review]

**Comment by gewarren** [comment]
@mairaw This one is preapproved by Maoni if you could take a quick look.

**Review by mairaw — APPROVED** [review]
LGTM. Would this warrant an ms.date update? Also, I've noticed that the acronym GC was introduced several times in this doc. Can you just say GC on the subsequent introductions?

**Pull request merged**